client-js/app/elements/labrad-plot.ts

```diff
@@ -95,6 +95,9 @@ export class Plot extends polymer.Base {
   @property({type: Number, value: 0})
   numIndeps: number;
 
+  @property({type: String, value: ''})
+  currPos: string;
```

Comment:
I don't want to put string formatting code in the template, so the string has to be assembled here and passed to the template as a string. You could pass numbers, but then you'd get the same 15-digit default formatting that you've already pointed out is kind of annoying.